utils/streamlit_utils.py

```diff
@@ -203,60 +253,148 @@ def expander_text_lengths(dstats, column_id=""):
             )
 
 
-def expander_text_duplicates(dstats, column_id=""):
-    with st.expander(f"Text Duplicates{column_id}", expanded=False):
-        st.caption(
-            "Use this widget to identify text strings that appear more than "
-            "once."
+
+
+### Third, use a sentence embedding model
+#TO:DO delete
```

Comment:
This seems to be from an older version of the file -- text embeddings are removed in the newer versions. It would be good to merge in the update streamlit_utils.py if possible (which has this removed and other changes), but this should be removed either way.